tapir/utils/management/commands/populate_functions.py

```diff
@@ -30,25 +31,39 @@ def populate_users():
 
     parsed_users = get_test_users()
     for index, parsed_user in enumerate(parsed_users[:USER_COUNT]):
-        if index % 50 == 0:
+        if index % 10 == 0:
             print(str(index) + f"/{USER_COUNT}")
         json_user = JsonUser(parsed_user)
-        randomizer = index + 1
-
-        is_company = randomizer % 70 == 0
-        is_investing = randomizer % 7 == 0 or is_company
-
-        tapir_user = None
-        if not is_company and not is_investing:
-            tapir_user = TapirUser.objects.create(
-                username=json_user.get_username(),
-            )
-            copy_user_info(json_user, tapir_user)
-            tapir_user.is_staff = False
-            tapir_user.is_active = True
-            tapir_user.date_joined = json_user.date_joined
-            tapir_user.password = tapir_user.username
-            tapir_user.save()
+
+        # randomizer = index + 1
+        # is_company = randomizer % 70 == 0
+        # is_investing = randomizer % 7 == 0 or is_company
+
+        # wirgarten_user = None
+        # if not is_company and not is_investing: # remove
+        wirgarten_user = Member.objects.create(
```

Comment:
Why the INSERT statement and then the UPDATE statement? You can just insert the complete data directly? The correct way would be to use the Member() constructor and pass the instance to the `service.member.create_member()` function.